src/shared/components/MapLegendLoadUnloadSpaces.tsx

```diff
@@ -43,7 +43,7 @@ export const MapLegendLoadUnloadSpaces = ({ checked, onChange }: MapLegendLoadUn
   return (
     <StyledLabel disabled={isDisabled} htmlFor="mapLegendLoadUnloadSpaces" label={loadUnloadSpaces.label}>
       <Legend />
-      <Checkbox id="mapLegendLoadUnloadSpaces" onChange={onChange} checked={checked} />
+      <Checkbox id="mapLegendLoadUnloadSpaces" onChange={onChange} checked={!isDisabled && checked} />
```

Comment:
Sanitycheck:
1. Does this visually show the checkbox is disabled?
2. Does the onChange change the value of checked even though the checkbox is disabled? This shouldn't be the case right?

Reply:
1. Yes
2. It does not because a disabled checkbox does not fire a change event.